src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1907,62 +1826,75 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         Set<String> studentsEmailInTeam = new HashSet<String>();
         if (role == UserRole.STUDENT) {
             student = studentsLogic.getStudentForEmail(courseId, userEmail);
-            List<StudentAttributes> studentsInTeam = studentsLogic
-                    .getStudentsForTeam(student.team, courseId);
+            List<StudentAttributes> studentsInTeam = studentsLogic.getStudentsForTeam(student.team, courseId);
             for (StudentAttributes teammates : studentsInTeam) {
                 studentsEmailInTeam.add(teammates.email);
             }
         }
-        
+
         InstructorAttributes instructor = null;
         if (role == UserRole.INSTRUCTOR) {
             instructor = instructorsLogic.getInstructorForEmail(courseId, userEmail);
         }
         
         Map<String, FeedbackResponseAttributes> relevantResponse = new HashMap<String, FeedbackResponseAttributes>();
```

Comment:
Almost the whole method :p It's like loop over questions, add questions, loop over questions, add responses, loop over.... so on and so forth, it might be nicer if we can split up the responsibilities!